tests/unit/components/HapticCopy.spec.js

```diff
@@ -1,5 +1,5 @@
 import { mount } from '@vue/test-utils'
-import HapticCopy from '@/components/HapticCopy.vue'
+import HapticCopy from '@lib/components/HapticCopy.vue'
```

Comment:
Somewhere (sorry it's vague) I saw this component imported with relative path (looked like something like that: `import HapticCopy from './HapticCopy.vue'` )

It could be great to change to look like this `import HapticCopy from '@lib/components/HapticCopy.vue'`

Reply:
The imports are not consistent indeed. We should do a refactoring branch for that purpose. Maybe with additional alias for keys directories (`@components`, `@maps`, etc)?